pkg/kubelet/network/cni/cni.go

```diff
@@ -53,24 +59,24 @@ type cniNetwork struct {
 
 func probeNetworkPluginsWithVendorCNIDirPrefix(pluginDir, vendorCNIDirPrefix string) []network.NetworkPlugin {
 	configList := make([]network.NetworkPlugin, 0)
-	network, err := getDefaultCNINetwork(pluginDir, vendorCNIDirPrefix)
-	if err != nil {
-		return configList
-	}
-	return append(configList, &cniNetworkPlugin{
-		defaultNetwork: network,
+	plugin := &cniNetworkPlugin{
+		defaultNetwork: nil,
 		execer:         utilexec.New(),
-	})
+	}
+
```

Comment:
add some comments like:

sync network config from pluginDir periodically to detect network config updates.